application/clicommands/ProcessCommand.php

```diff
@@ -72,6 +72,7 @@ public function listAction()
      *   --config <configname>   Name of the config that contains <process>
      *   --details               Show problem details as a tree
      *   --colors                Show colored output
+     *   --handled               Show global Acknowledge handle state
```

Comment:
Why are you mixing the terms "acknowledged" and "handled"?

Reply:
because it's known in the frontend as Acknowledge but the switch for the backend is named as handle. So to avoid confusion of our employees I named it Acknowledge. But feel free and change it or tell me what is your prefered.

Reply:
`acknowledged` and `handled` are two different things, with `acknowledged` being a subset of `handled`. While an `acknowledged` object is always considered being `handled`, also objects `in_downtime` and services on a host NOT being `up` are considered `handled`.